webapp/channels/src/components/admin_console/access_control/modals/user_sync/user_sync_modal.scss

```
@@ -0,0 +1,60 @@
+.user-list-modal-dialog {
+    width: 660px;
+    max-width: 90vw;
+    padding: 0;
+
+    .modal-body {
+        max-height: fit-content;
+        padding: 0;
+    }
+
+    .col-sm-12 {
+        padding-top: 12px;
+        padding-bottom: 12px;
+        padding-left: 0;
+    }
+
+    .col-xs-12 {
+        padding-right: 0;
+        padding-left: 0;
+    }
+
+    .more-modal__row {
+        padding: 10px 0;
+    }
+}
+
+.tabs {
+    display: flex;
+    padding: 0 16px;
+    border-bottom: 1px solid var(--center-channel-color-08);
```

We've been trying to remove these numbered CSS variables for a while now. Could you change these to use the `-rgb` ones? In this case, it'd be
```suggestion
    border-bottom: 1px solid rgba(var(--center-channel-color-rgb), 0.8);
```
and there's a few others in this file as well